VideoWeb/VideoWeb/ClientApp/src/index.html

```diff
@@ -19,13 +19,145 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <script>
```

Comment:
This should navigate to the not supported page rathe than duplicate the whole page